.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,36 +1,18 @@
-# Required for all PRs
-
-<!-- Before opening a pull request you should run the following checks locally to make sure the CI will pass.
-
-make lint
-make check
-make check-deps
-make test
-make docs
-
+## Summary
+<!-- Mandatory
+Explain here the why, the rationale and motivation, for the changes.
 -->
 
-<!-- Complete the tasks in the following list. Change [ ] to [x] to
-show completion. -->
-
-- [ ] Updated associated README.md.
-- [ ] Wrote appropriate unit tests.
-- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)
-
-<!-- Link to issues that describe the need for the change. Issues
-should include context that will help reviewers understand why the
-change is needed.
-
-Make sure to link issues and using a keyword like "resolves #1234".
-https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+## Checklist
+<!-- Mandatory
+Please confirm the following by placing an "x" between the []:
 -->
 
-resolves #
+- [ ] No AI generated code was used in this PR
 
-<!-- Finally, include a summary of the code change itself. This
-description should tell reviewers how the issues were resolved.
-
-example: Fixed an off by one error in counter variable in type FooBar.
+## Related issues
+<!-- Mandatory
+All PRs should resolve an issue, if one does not exist, please open one.
+-->
 
-example: Added an input plugin to gather yak shaving metrics using
-golang library yaktech/shaver. -->
+fixes #
```

Comment:
Can we please keep `resolves`? It reads strange that we "fix" a feature request. :-)

Reply:
done!